### DISCUSSION TITLE
ios in app purchases add localization

### DISCUSSION THREAD

**Discussion by nasru** [fastlane/fastlane#20346] [General]
Hello there. I am trying fastlane for the first time. very practical and beautiful. Thank you to those who contributed. I made the settings in the console automatically. All my metadata has arrived. but in app purchases localization data is not received. How can I update my in app purchases data or add new languages?

this is my fastfile :

default_platform(:ios)

platform :ios do
  desc "Push a new release build to the App Store"
  lane :release do
    increment_build_number(xcodeproj: "WordEst.xcodeproj")
    build_app(workspace: "WordEst.xcworkspace", scheme: "WordEst")
    upload_to_app_store
  end
end

**Comment by 954708775** [comment]
Same waiting for a solution here
